src/components/ChangePlan/PlanCalculator/Discounts/Discounts.js

```diff
@@ -18,34 +22,57 @@ export const Discounts = ({ discountsList, handleChange }) => {
     }
   };
 
+  useEffect(() => {
```

Comment:
what is the handleChange function used for? 
Every time a discount change occurs, 2 functions are executed: handleChange and setSelectedDiscount. It may be a good idea to have a single function that receives the discount to change and that invokes these 2 functions